File: PgAce.App/Forms/DbExplorerPane.vb
Imports WeifenLuo.WinFormsUI.Docking

Namespace PgAce.App.Forms
    Public Class DbExplorerPane
        Inherits DockContent

        Public Sub New()
            Text = "DB Explorer"
            DockAreas = DockAreas.DockLeft Or DockAreas.DockRight Or DockAreas.Float
        End Sub
    End Class
End Namespace

File: PgAce.App/Forms/DbExplorerPane.cs
using WeifenLuo.WinFormsUI.Docking;

namespace PgAce.App.Forms
{
    public class DbExplorerPane : DockContent
    {
        public DbExplorerPane()
        {
            Text = "DB Explorer";
            DockAreas = DockAreas.DockLeft | DockAreas.DockRight | DockAreas.Float;
        }
    }
}
